var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -269,19 +269,20 @@ def install_test(self):
                 )
 
             # Spack's logs don't handle colored output well
-            bazel = Executable(self.prefix.bin.bazel)
+            bazel = Executable(self.spec["bazel"].command.path)
             bazel(
                 "--output_user_root=/tmp/spack/bazel/spack-test",
                 "build",
                 "--color=no",
+                f"--jobs={make_jobs}",
                 "//:bazel-test",
             )
 
             exe = Executable("bazel-bin/bazel-test")
             assert exe(output=str) == "Hi!\n"
 
     def setup_dependent_package(self, module, dependent_spec):
-        module.bazel = Executable("bazel")
+        module.bazel = Executable(self.spec["bazel"].command.path)
```

Comment:
What's wrong with `self.command.path`?

Reply:
I'm confused, it's not implemented?

Reply:
Ah okay, SpecBuildInterface... that's very ugly :weary:

Reply:
Did you want to turn this into a `MakeExecutable`?

```
        module.make = MakeExecutable(
            ..., determine_number_of_jobs(parallel=dependent_spec.package.parallel, supports_jobserver=False)
        )
```

Reply:
I tried this at first but it doesn't work because it needs to be `bazel build -j`, not `bazel -j build`.